src/lib/cfa/ntp_conf.rb

```diff
@@ -328,8 +346,8 @@ def initialize
     #
     # Fudge entry has its own options interpretation.
     class FudgeRecord < CommandRecord
-      def initialize
-        super("fudge[]")
+      def create_augeas
+        { key: "fudge[]", value: nil }
```

Comment:
well, when I see this method overwritten it almost always look same and from 
```
def create_augeas
  { key: "fudge[]", value: nil }
end
```

only information holder it "create_augeas" so we know what is doing and "fudge[]" which doesn't look like enough for me and too much non-interesting stuff there. And also it is not instance specific, but class specific.

So what about using something like 
```
class FudgeRecord
  AUGEAS_KEY = "fudge[]"
  ...
end

# and in Record
def create_augeas
  { key: self.class.const_get("AUGEAS_KEY"), value: nil }
end
```

another option is class attribute or class method. I am fine with any of it which simplify kids of Record to hold only relevant information.